IS7024Project/Pages/Index.cshtml

```diff
@@ -98,34 +99,16 @@
                 <tr>
                     <td>@crime.IncidentNo</td>
                     <td>@crime.Offense</td>
-                    <td>@crime.Location</td>
+                    <td>@Regex.Replace(@Regex.Replace(@crime.Location.ToString(), "[\\d-]", string.Empty), "([a-z])([A-Z])", "$1 $2")</td>
                     <td>@crime.DateFrom</td>
                     <td>@crime.Dayofweek</td>
-                    <td>@crime.CommunityCouncilNeighborhood</td>
-                    <td>@crime.CpdNeighborhood</td>
+                    <td>@Regex.Replace(@crime.CommunityCouncilNeighborhood.ToString(), "([a-z])([A-Z])", "$1 $2")</td>
+                    <td>@Regex.Replace(@crime.CpdNeighborhood.ToString(), "([a-z])([A-Z])", "$1 $2")</td>
                     <td>@crime.LatitudeX</td>
                     <td>@crime.LongitudeX</td>
                     <td>@crime.Zip</td>
                 </tr>
             }
         }
     </table>
-    <div style="padding-top: 20px">
```

Comment:
cut this table to put on a new page for better content management

Reply:
Is it an entirely new page?  Good usability == minimized clicks == minimized pages.  So, if it requires an extra click to see the content now, I'd lean toward fewer clicks.